### PR TITLE
Answer_it refine

### PR DESCRIPTION
**Pull request by alterxyz**
- fix: cohere long answer and garbled code
- fix: update same message

### Answer_it

- refactor: Modulated
- chore: Switch on the top as "Customization"
- chore: Prompt for tasks
- feat: Support more LLM as optional
- feat: Summarization for final answer
- feat: Asynchronous/Thread for faster speed
- feat: Background Run some LLM, no telegram display, append to telegra.ph instead

### CODE REVIEW

File: handlers/useful.py [hunk 1]
```diff
@@ -172,85 +201,205 @@ def answer_it_handler(message: Message, bot: TeleBot):
...
+        chatgpt_future = executor.submit(chatgpt_answer, latest_message, bot, m)
+    if COHERE_USE and COHERE_API_KEY:
+        cohere_future = executor.submit(cohere_answer, latest_message, bot, m)
+    if QWEN_USE and QWEN_API_KEY:
+        qwen_future = executor.submit(qwen_answer, latest_message, bot, m)
+    if CLADUE_USE and ANTHROPIC_API_KEY:
+        claude_future = executor.submit(claude_answer, latest_message, bot, m)
+    if LLAMA_USE and LLAMA_API_KEY:
+        llama_future = executor.submit(llama_answer, latest_message, bot, m)
+
```
Comment (yihong0618) [inline handlers/useful.py:222]: do not pass the bot for threading, for tg bot, only one bot instance at the time, you can check the log

Comment (alterxyz) [reply handlers/useful.py:222]: What would happen?

Comment (alterxyz) [reply handlers/useful.py:222]: May I know why?

Comment (alterxyz) [reply handlers/useful.py:222]: I believe we can implement bot threading without issues, as my testing has shown no problems.

I've recently added a new feature that skips message updates if the old and new messages are identical. This has significantly reduced log entries.

It appears that telebot supports some kinda asynchronous operations, which means we can have multiple "bot" and pass the "bot " as an object.

I noticed an issue with group messaging: when the LLM reply is exceptionally long and splits into multiple texts, the 'delete chat by ID' function doesn't catch the extra messages. However, this occurs rarely and isn't within the intended scope of the `answer_it` function, so we can consider it a low priority for now.

Please test the `/answer_it` command and other functions using this [bot](https://t.me/tealmoon_ai_v1_alt_bot)

**Comment by yihong0618** [comment]
nice work

**Pull request merged**